src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../type/User';
+
+interface Props {
+  user: User | null;
```

Comment:
if `null` is the value then the link will be empty
therefore, it is better to require that the user always be an object
so that this component can be used like this
`{user && <UserInfo user={user} />}`
or if you still leave the value as `null` then better write this:
```
if (!user) {
  return null;
}

const { email, name } = user;

return (
  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
);
```